src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/DataPersisterPass.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Registers data persisters.
+ *
+ * @internal
```

Comment:
We'll be able to drop this class when we'll upgrade to DI 3.4: http://symfony.com/blog/new-in-symfony-3-4-simpler-injection-of-tagged-services

Reply:
Yep, I saw that. Do you suggest to delete it?